html_app/ui/microscopy.gss

```diff
@@ -1368,6 +1368,7 @@ top: 49px;
 	bottom: -17px;
 	overflow:hidden;
 	background:black;
+	cursor: pointer;
```

Comment:
Here you can see what it looks like: http://www.w3schools.com/cssref/tryit.asp?filename=trycss_cursor